pdks/Sky130/sky130_hdl21/pdk_data.py

```diff
@@ -67,7 +67,7 @@ class MosParams:
     w (h.Scalar): Width of the MOSFET in PDK Units (µm). Default is 650 * MILLI.
     l (h.Scalar): Length of the MOSFET in PDK Units (µm). Default is 150 * MILLI.
     nf (h.Scalar): Number of fingers in the MOSFET. Default is 1.
-    m (h.Scalar): Multiplier for the MOSFET (alias for mult). Default is 1.
+    mult (h.Scalar): Multiplier for the MOSFET. Default is 1.
```

Comment:
So I don't think "replace all `m` with `mult`" is gonna work.  
Particularly for parameters like this, which are used by `ExternalModule`s and are ultimately feed directly into external netlists.

Reply:
Ah right, these are used directly in netlisting? Thanks for pointing that out; I got a bit carried away here.

Reply:
Yeah, if memory serves, this is a quirk of the Sky130 PDK and it will throw an error unless it gets its specific `mult` or `m` parameter depending on the subcircuit you're using in the PDK.